website/static/js/profile.js

```diff
@@ -577,6 +598,44 @@ var ListViewModel = function(ContentModel, urls, modes) {
         }
         return true;
     });
+
+    self.institutionsEmpty = ko.computed(function() {
+        for (var i=0; i<self.contents().length; i++) {
+            if (self.contents()[i].institutionEmpty()) {
+                return true;
+            }
+        }
+        return false;
+    });
+
+   // Add enable/disable button css to use bootstrap and conform to style guide.
+   self.enablebtn = ko.computed(function(){
```

Comment:
For consistency, this should be `enabledBtn`.

Ideally, I'd prefer it to be something like `isDirty` or `formChanged` - something descriptive of what the variable represents, as opposed to its impact.